crates/ruff_linter/src/rules/pydocstyle/rules/blank_before_after_class.rs

```diff
@@ -197,7 +197,7 @@ pub(crate) fn blank_before_after_class(checker: &Checker, docstring: &Docstring)
                 // Delete the blank line before the class.
                 diagnostic.set_fix(Fix::safe_edit(Edit::deletion(
                     blank_lines_start,
-                    docstring.start() - docstring.indentation.text_len(),
+                    docstring.start() - docstring.indentation().text_len(),
```

Comment:
Nit: Consider a `indentation_len` method? Considering that most call sites aren't interested in the `indentation` itself (avoids going from range to string and back to range again.